packages/rx_bloc_cli/mason_templates/bricks/feature_login/__brick__/lib/feature_login/views/login_page.dart

```diff
@@ -75,23 +79,37 @@ class LoginPage extends StatelessWidget {
                     ],
                   ),
                 ),
-                Text.rich(
-                  textAlign: TextAlign.center,
-                  TextSpan(
-                    text: context.l10n.featureLogin.dontHaveAccount,
-                    style: context.designSystem.typography.h2Reg16
-                        .copyWith(color: context.designSystem.colors.gray),
-                    children: [
-                      const TextSpan(text: ' '),
-                      TextSpan(
-                        text: context.l10n.featureLogin.signUpLabel,
-                        style: const TextStyle(fontWeight: FontWeight.bold),
+                RxBlocBuilder<LoginBlocType, bool>(
+                  state: (bloc) => bloc.states.isLoading,
+                  builder: (context, isLoading, bloc) => Material(
```

Comment:
Do you think that the sign-up button deserves its own widget in the feature_login/ui_components ?

Reply:
Moved to `feature_login/ui_components/sign_up_button.dart`.